src/views/Login.vue

```diff
@@ -166,17 +168,22 @@ export default {
           );
 
           if (element && element.split(' ').length === 12) {
+            pasting.value = true;
+
             inputs.value = element.split(' ').map(el => ({ value: el }));
 
             try {
               await validateAllInputs();
             } catch (e) {}
 
             lastInput.focus();
+
+            pasting.value = false;
           } else if (element && element.includes(' ')) {
             const nextInput = document.getElementById(`passphrase-${i + 1}`);
             if (nextInput) {
               nextInput.focus();
+              setTimeout(() => (nextInput.value = ''), 50);
```

Comment:
Why does it require a 50 millisecond timeout?

Reply:
Can this be `0`? Instead of `50`. Usually to get something to run on the 'next tick', setTimeout 0 is used.

Reply:
See commit 795f2a60ae9240f2b9e4d8a770f3d3785717d5d7 where I used Vue's `nextTick` solution